### PR TITLE
[ENH] OWMultiSample: Add new widget

### LINKED ISSUE #142: multi-sample loader
Need a way to load multiple files at once

* select multiple files - all files need to be in the same format
* merge all samples into single matrix with labels

### PR DESCRIPTION
**Pull request by VesnaT**
##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

Fixes #142

### CODE REVIEW

**Comment by BlazZupan** [comment]
Please include the support for pickled and zipped input files as saved by Orange. See http://file.biolab.si/files/tmp/P1.pkl.xz for example of such file.

**Comment by BlazZupan** [comment]
When displaying number of genes and columns, please take into consideration the choice of the input data structure (genes in rows or cells in rows).

**Comment by lanzagar** [comment]
Also, add *.tab as a filter for tab-delimited files in addition to *.tsv

**Comment by VesnaT** [comment]
Pickled and compressed files will be supported when #223 is merged.

File: orangecontrib/single_cell/widgets/owmultisample.py [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+    def commitData(self, event):
+        super().commitData(event)
+        self.data_changed.emit()
+
+
+class OWMultiSample(owloaddata.OWLoadData):
+    name = "Multi Sample"
+    description = "Load samples for multi-sample analysis"
+    icon = "icons/MultiSample.svg"
+    priority = 200
```
Comment (lanzagar) [inline orangecontrib/single_cell/widgets/owmultisample.py]: The priority should be lower, placing this widget somewhere at the start - probably right after SC Datasets.

**Pull request merged**